GDJS/Runtime/gd.js

```diff
@@ -98,19 +98,35 @@ gdjs.toDegrees = function(angleInRadians) {
  * The name of the type of the object must be complete, with the namespace if any. For
  * example, if you are providing a Text object in the TextObject extension, the full name
  * of the type of the object is "TextObject::Text".
+ *
+ * @deprecated
  */
 gdjs.registerObjects = function() {
   gdjs.objectsTypes.clear();
 
   for (var p in this) {
     if (this.hasOwnProperty(p)) {
-      if (gdjs[p].thisIsARuntimeObjectConstructor != undefined) {
-        gdjs.objectsTypes.put(gdjs[p].thisIsARuntimeObjectConstructor, gdjs[p]);
+      if (gdjs[p].thisIsARuntimeObjectConstructor !== undefined) {
+        gdjs.registerObject(gdjs[p], gdjs[p].thisIsARuntimeObjectConstructor);
       }
     }
   }
 };
 
+/**
+ * Register the runtime objects that can be used in runtimeScene.<br>
+ *
+ * The name of the type of the object must be complete, with the namespace if any. For
+ * example, if you are providing a Text object in the TextObject extension, the full name
+ * of the type of the object is "TextObject::Text".
+ *
+ * @param Ctor The constructor of the Object.
+ * @param objectTypeName The name of the type of the Object.
+ */
+gdjs.registerObject = function(Ctor, objectTypeName){
```

Comment:
Same as behavior, let's be swap the parameter order to hint the reader that registerObject is actually putting things in a map: `function(objectTypeName, Ctor)`